docs/fio.md

```diff
@@ -0,0 +1,263 @@
+# FIO I/O  Workload
+
+The FIO I/O workload playbook is `workloads/fio.yml` and will run the FIO I/O workload on your cluster.
+FIO I/O workload test is designed to stress storage backend used by
+application pods. FIO I/O workload test supports all FIO test types : seq read /
+write, random read/write, randrw, rw.
+
+
+Requirements:
+
+* Working storage backend and working storage class 
+
+FIO I/O test will use dynamic provisioning to allocate PVC and attach it to Pod
+so it requirement to have working storage class prior starting this test.
+
+* `PBENCH_SERVER` variable to set to valid pbench server 
+
+How to setup pbench server is described in pbench
+[documentation](https://distributed-system-analysis.github.io/pbench/doc/server/installation.html)
+
+ 
+Running from CLI:
+
+```sh
+$ cp workloads/inventory.example inventory
+$ # Add orchestration host to inventory
+$ # Edit vars in workloads/vars/nodevertical.yml or define Environment vars (See below)
```

Comment:
s/nodevertical/fio

Reply:
This still appears to be "workloads/vars/nodevertical.yml" for me